cmds/km/main.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2017-2018 the LinuxBoot Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// fspinfo prints FSP header information.
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/linuxboot/fiano/pkg/intel/metadata/cbnt/cbntkey"
+	"github.com/linuxboot/fiano/pkg/log"
+)
+
+var (
+	flagJSON = flag.Bool("j", false, "Output as JSON")
+)
+
+func main() {
+	flag.Parse()
+	if flag.Arg(0) == "" {
+		log.Fatalf("missing file name")
+	}
+	data, err := os.ReadFile(flag.Arg(0))
+	if err != nil {
+		log.Fatalf("cannot read input file: %v", err)
+	}
+
+	KEYMMagic := []byte("__KEYM__")
+	offset := bytes.Index(data, KEYMMagic)
```

Comment:
if you only want to check if bytes are here bytes.Contains are better and you can get rid of offset, or maybe you forgot to read `data` from offset?
```go
	if !bytes.Contains(b, []byte("bytes")) {
		log.Fatal("...")
	}
```